modin/pandas/test/test_io.py

```diff
@@ -1387,6 +1387,21 @@ def test_read_parquet_without_metadata(self):
         finally:
             teardown_test_files([parquet_fname, csv_fname])
 
+    @pytest.mark.xfail(
+        condition="config.getoption('--simulate-cloud').lower() != 'off'",
```

Comment:
Does the CI fail if we don't include this xfail?

Reply:
Nope, I deleted it.